c10/test/util/complex_test_common.h

```diff
@@ -167,6 +169,11 @@ TEST(TestConstructors, FromThrust) {
 }
 #endif
 
+TEST(TestConstructors, UnorderedMap) {
+  std::unordered_map<c10::complex<double>, c10::complex<double>> m;
+  m[c10::complex<double>(2.5, 3)] = c10::complex<double>(2, -3.2);
+  m[c10::complex<double>(2, 0)] = c10::complex<double>(0, -3);
```

Comment:
This isn't a particularly robust test haha

Reply:
lol yeah I updated the test to access values and check equality